docs/pip-requirements.txt

```diff
@@ -0,0 +1,3 @@
+mkdocs-material
+mike
+mkdocs-redirects==1.2.1
```

Comment:
Should we have a version of minify included here ? I get the error
```
ERROR   -  Config value 'plugins': The "minify" plugin is not installed
```
going through a clean setup

Reply:
I will add it